docs/user/config/babelConfig.md

```diff
@@ -2,11 +2,11 @@
 title: Babel Config option
 ---
 
-TSJest by default does **NOT** use Babel. But you may want to use it, especially if your code rely on Babel plugins to make some transformations. TSJest can call the BabelJest processor once TypeScript has transformed the source into JavaScript.
+ts-jest by default does **NOT** use Babel. But you may want to use it, especially if your code rely on Babel plugins to make some transformations. ts-jest can call the BabelJest processor once TypeScript has transformed the source into JavaScript.
```

Comment:
BabelJest -> babel-jest. I think we should put in backtick as well for ts-jest

Reply:
well, that's the thing about my use of TSJest over `ts-jest`. @GeeWee said you had a big discussion about it and I'd love to know the pros and cons you ended up with.

I used `TSJest`, `BabelJest` to name the tools, while using `ts-jest` and `babel-jest` to name specifically the npm packages. Was also to match better things like TypeScript, Babel, ...

I'm ok to use `ts-jest` and `babel-jest` to name the packages and tools, but then we must replace TypeScript with typescript, Babel with babel, BabelJest with babel-jest, ...

Reply:
I feel like some projects have actual names, like Typescript and Babel, but some packages don't? I think maybe it's a popularity thing.
I remember the discussion as longer, but we agreed on the ts-jest naming here https://github.com/kulshekhar/ts-jest/issues/273

Reply:
It looks like it's not about the naming in #273, it's the font style to use on chosen `ts-jest`. I meant, it sounds to me there is nothing about TSJest vs ts-jest.

Reply:
That's true. Maybe merge this in and then open up the discussion seperately? I've gotta dash soon.